### PR TITLE
Update ZeroFox.py

### PR DESCRIPTION
**Pull request by zf-jedmunds**
Updating ZeroFox API parameter to allow for fetching updated alerts and not just net new alerts. The current parameter being passed to our (ZeroFox) API only allows for users of this integration to receive net new alerts and not alerts which have been updated by our platform, staff or other customer teams.

old parameter: min_timestamp
new parameter: last_modified_min_date

Please let me know if you have any questions.

Jon
jedmunds@zerofox.com

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @zf-jedmunds, 
Thank you for contributing!

In order to complete your contribution please create a release-note file with an explanation of the change you did in the integration, and bump the pack's version in accordance - 
You can easily do so by running the  `demisto-sdk update-release-notes -i Packs/ZeroFox` command.
For more information see also: [update-release-note](https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes).

Just out of curiosity, was it an API change that replaced the 'min_timestamp' key with 'last_modified_min_date'? 

Please don't hesitate to contact me with any questions :) 
(I'm available here or via slack).

**Comment by zf-jedmunds** [comment]
@ShacharKidor this update was not related to an API change - we've always leveraged last_modified_min_date. I think given that we've had an increasing number of customers using XSOAR that the issue finally got brought to my attention.

File: Packs/ZeroFox/ReleaseNotes/1_0_5.md
Comment (ShacharKidor) [inline Packs/ZeroFox/ReleaseNotes/1_0_5.md:3]:
```suggestion
- Updated the **ZeroFox** API call to **leverage last_modified_min_date** to fetch all alerts with updates.
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @zf-jedmunds - good job.

**Comment by ShacharKidor** [comment]
@zf-jedmunds - In order to solve the CircleCI build's errors please: 

1. Create a readme file for the ZeroFox integration - you can use the: 
`demisto-sdk generate-docs -i Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.yml` command for that. 

2. Update the docker image tag in the ZeroFox.yml to the latest tag (3.10.4.30607).

Please contact me if you have any questions :)

**Comment by zf-jedmunds** [comment]
@ShacharKidor added readme and updated docker

**Comment by zf-jedmunds** [comment]
@ShacharKidor - any updates on this?

**Comment by ShacharKidor** [comment]
> @ShacharKidor - any updates on this?

@zf-jedmunds - Still waiting for the approval of the documentation review - @jrosenpanw Can you please assign this PR to a technical writer?

@zf-jedmunds - In the meantime, please merge master (the build is failing because the pr is not up to date with the latest changes).

**Comment by ShirleyDenkberg** [comment]
@ShacharKidor Doc review completed.

File: Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.yml
Comment (ShacharKidor) [inline Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.yml:7]:
```suggestion
  display: URL (e.g. https://api.zerofox.com/1.0)
```

File: Packs/ZeroFox/ReleaseNotes/1_0_5.md
Comment (ShacharKidor) [inline Packs/ZeroFox/ReleaseNotes/1_0_5.md:3]:
```suggestion
#### Integrations
##### ZeroFox
- Updated the **ZeroFox** API call to leverage *last_modified_min_date* to fetch all alerts with updates.
- Updated the Docker image to: *demisto/python3:3.10.4.30607*.
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @zf-jedmunds, 
I requested two last little changes - please see them below. 
In addition, please merge from mater so we can merge your PR. 

Thank you!

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**
